core/src/main/java/org/eclipse/hono/connection/ConnectionFactoryImpl.java

```diff
@@ -127,14 +127,14 @@ private void handleConnectionAttemptResult(
 
         if (conAttempt.failed()) {
 
-            logger.debug("can't connect to AMQP 1.0 container [{}://{}:{}]: {}", clientOptions.isSsl() ? "amqps" : "amqp",
+            logger.warn("can't connect to AMQP 1.0 container [{}://{}:{}]: {}", clientOptions.isSsl() ? "amqps" : "amqp",
```

Comment:
this is logged at DEBUG level intentionally since this case is handled by continuously re-trying to connect
however, if e.g. the peer is not (yet) up and running, the log is effectively polluted by these (meaningless) messages

Reply:
this also doesn't seem to be related to the changes in the JMeter plugin and thus should be made in a different PR

Reply:
I agree.